scripts/settlement/register_sequencer_to_hub.sh

```diff
@@ -4,14 +4,17 @@ EXECUTABLE="rollapp-evm"
 KEYRING_PATH="$HOME/.rollapp_evm/sequencer_keys"
 KEY_NAME_SEQUENCER="sequencer"
 
-
 #Register Sequencer
-DESCRIPTION="{\"Moniker\":\"myrollapp-sequencer\",\"Identity\":\"\",\"Website\":\"\",\"SecurityContact\":\"\",\"Details\":\"\"}";
+DESCRIPTION="{\"Moniker\":\"${ROLLAPP_CHAIN_ID}-sequencer\",\"Identity\":\"\",\"Website\":\"\",\"SecurityContact\":\"\",\"Details\":\"\"}"
 SEQ_PUB_KEY="$($EXECUTABLE dymint show-sequencer)"
+BOND_AMOUNT="100000dym"
```

Comment:
it's too high bond, more that u fund the sequencer in the readme.
u can use the default gensis value for `min_bond`